utils/index.sh

```
@@ -2,6 +2,34 @@
+        chmod +x "${OCP_METADATA_TOOL}"
+    fi
+
+    # Run ocp-metadata and capture output as JSON
+    OCP_METADATA_JSON=$(./${OCP_METADATA_TOOL})
+
+    # Export the JSON for later use in index_task()
+    export OCP_METADATA_JSON
+
+    # Extract RELEASE_STREAM for setup function
+    cluster_version=$(echo "$OCP_METADATA_JSON" | jq -r '.ocpVersion // ""')
+    export RELEASE_STREAM=$(echo "$cluster_version" | cut -d '-' -f1-2)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)